### PR TITLE
Update cross-tenant-mailbox-migration.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed mispelling

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7454

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ To obtain the tenant ID of a subscription, sign in to the [Microsoft 365 admin c
 
 12. Now we need to add permission for mailbox migration, select Add a permission
 
-13. In the Request API permissions windows, select APIs my organization users, and search for office 365 exchange online, select it.
+13. In the Request API permissions windows, select APIs my organization uses, and search for office 365 exchange online, select it.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
13. In the Request API permissions windows, select APIs my organization uses, search for Office 365 Exchange Online, and select it.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**